### PR TITLE
Eligibility cron

### PR DESCRIPTION
**Pull request by parthgandhitekdi**
Task: [Create script to update the eligibility for all the users](https://tracker.tekdi.net/issues/201508)

### CODE REVIEW

File: src/src/cron/calculateEligibility.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+		let userIds = (
+			await this.hasuraService.getData({ query: hasuraQuery })
+		).data.users.map((user) => user.id);
+
+		// Loop through them and update eligibility details
+		for (const id of userIds) {
+			try {
+				await this.facilitatorService.updateEligibilityDetails(id);
+			} catch (error) {
```
Comment (yogesh-pawar-2023) [inline src/src/cron/calculateEligibility.service.ts:41]: remove console.log()

File: src/src/cron/calculateEligibility.service.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+				console.log(
+					'\n******************** Error while updating eligibility details for the userid:',
+					id,
+					'********************',
+				);
+				console.log(error);
+				console.log(
```
Comment (yogesh-pawar-2023) [inline src/src/cron/calculateEligibility.service.ts:48]: remove console.log()

File: src/src/cron/faceIndexing.service.ts [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+
+			// Step-1: Fetch all users whose fa_user_indexed value is false or null.
+			const usersToIndexFaces = await this.fetchAllUsersToIndexFaces(
+				parseInt(
+					this.configService.get<string>(
+						'AWS_REKOGNITION_INDEX_USER_BATCH_SIZE',
+					),
+				),
+			);
+			console.dir(usersToIndexFaces, { depth: 99 });
```
Comment (yogesh-pawar-2023) [inline src/src/cron/faceIndexing.service.ts:229]: remove console.log

**Comment by manojLondhe** [comment]
Closing unmerged - lets reuse code in future if needed

cc @rushi-tekdi @sagar-techjoomla @Tusharmahajan12 @GitVivekHub @YoginiTayade

**Pull request closed without merging**